callBackPersistence.js

```diff
@@ -55,21 +55,11 @@ class CallBackPersistence extends CachedPersistence {
       this.once('ready', this.addSubscriptions.bind(this, client, subs, cb))
       return
     }
-
-    const addSubs1 = this.asyncPersistence.addSubscriptions(client, subs)
-    // promisify
-    const addSubs2 = new Promise((resolve, reject) => {
-      this._addedSubscriptions(client, subs, (err) => {
-        if (err) {
-          reject(err)
-        } else {
-          resolve()
-        }
-      })
-    })
-    Promise.all([addSubs1, addSubs2])
-      .then(() => cb(null, client))
+    this.asyncPersistence.addSubscriptions(client, subs)
```

Comment:
I usually call `promise().then().catch()` why did you put the catch in the middle here?

Reply:
I thought it would be more readable to as the .then is longer and the original callback code also started with `if (err)` . Semantically it should not make a difference, but happy to change them around if you prefer.

Reply:
@seriousme I think it would be better to have catch set as last, at least this is how I always saw it used even if it could work in the other way 🙏🏼

Reply:
No problem, just changed them around.